src/familiar.ts

```diff
@@ -270,7 +272,14 @@ const standardFamiliars: () => GeneralFamiliar[] = () =>
     },
     {
       familiar: $familiar`Robortender`,
-      expectedValue: 200,
+      expectedValue:
+        garboValue($item`elemental sugarcube`) / 5 +
+        (Robortender.currentDrinks().includes($item`Feliz Navidad`)
+          ? get("garbo_felizValue", 0) * 0.25
+          : 0) +
+        (Robortender.currentDrinks().includes($item`Newark`)
+          ? get("garbo_newarkValue", 0) * 0.25
+          : 0),
     },
     ...$familiars`Hobo Monkey, Cat Burglar, Urchin Urchin, Leprechaun`.map((familiar) => ({
```

Comment:
I would steal the code I put in here--it makes it so we only consider your meatFamiliar(), rather than every lep-like